seeders/20200206005427-demo-patient.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
         updatedAt: new Date(),
       },
       {
-        gender: 'unknown',
+        gender: 'Male',
```

Comment:
shouldn't we be using the new enum values now? (same goes for all other `gender` values in seeders)
```suggestion
        gender: 'M',
```